script/Ciel/Day 8/08_01_CIEL08_6.txt

```diff
@@ -1,173 +1,190 @@
 [sha:8f92f1eb7e57f7ae5cf87f3b53d232037322b67c]{
 -- Page 10014, Offset 19528.
 -- ……午前零時。誰もが寝静まった深夜、遠野志貴の部屋にこっそり戻ってきた。
-Midnight. In the dead-silent night, I sneak back into my room.
+It's midnight... In the dead of night, while everyone else is asleep, I tiptoe back to my room.
 }
 [sha:977d5592a6ae315b18d2f4356da599c2444e2462]{
 -- Page 10014, Offset 19529.
 -- 夜に出かけて戻ってくる、この工程も慣れたものだ。
-Sneaking out and returning at night is a routine I've grown accustomed to.
+I've already grown accustomed to sneaking out and back again during the night.
 }
 [sha:f548d3c9af753356870d735d3f195745095dc80c]{
 -- Page 503, Offset 19530.
 -- 寝間着に着替えてベッドに横になる。
-I change into my pajamas and lie on the bed.
+I change into my pajamas and get into bed.
 }
 [sha:cbfeb15fe44e4af859cf6cbb84d6a5a3ee822df3]{
 -- Page 503, Offset 19531.
 -- 途端、急速な眠気が襲ってきた。
-At that very instant, I'm overcome by a sudden drowsiness.
+As soon as I do, I'm overcome by a sudden drowsiness.
 }
 [sha:015430a1042e9f4a92384a48b56ae18022b8dea1]{
 -- Page 503, Offset 19532.
 -- 体はそこまで疲れていないものの、魔眼の使いすぎで脳が休憩を求めているのだろう。
-Though I don't feel particularly exhausted, I suppose my brain needs to recover from overusing my Mystic Eyes.
+My body isn't all that tired,
+ but I guess my brain needs a break after I went overboard using my Mystic Eyes.
 }
 [sha:1126dc8bfcc8d058eb033094925d0a783fe383eb]{
 -- Page 503, Offset 19533.
 -- 眼鏡を外し、枕もとに置いて目を閉じる。
-I remove my glasses, place them by the pillows and close my eyes.
+I remove my glasses, place them on the bedside table, and close my eyes.
 }
 [sha:65dddd93af20d188dc08de3cf22156980157bb24]{
 -- Page 504, Offset 19534.
 -- 「……………あれ。
-"...Mmm?
+"Huh...?
 }
 [sha:f86bcdd647c294cd048f6336ab02f25f2345c81f]{
 -- Page 504, Offset 19535.
 -- 何か、考え事があった気がするんだけど……」
-I feel like there was something weighing on my mind."
+I feel like I'm forgetting something..."
 }
 [sha:5cdc68d3a3bb8bb383c7b3d5b09c60ad4ac23987]{
 -- Page 505, Offset 19536.
 -- 今夜も眠気に負けてしまう。
 // i read this as morbius
+// Morpheus, Dorpheus, Orpheus / Go eat some walruses / Orifices, Porridges / Morpheus Morpheus
```

Comment:
Morpheus drinkin a 40 in a death basket

that said, are we sure we want to use this? i don't exactly think and as a result
1. i ain't exactly
2. I was like "huh? morpheus? what?" while the beautiful face of lawrence fishburne flashed before my eyes. 


Sure I eventually trudged up some thought to figure out what this meant, but for all the non-thinkers out there like myself, my vote is for
> Tonight, I'll surrender myself to sleep.

Reply:
It kinda like it, but I suppose its one of those times when one ought to remember that this is a translation, and there is little to justify in the original text for such a sentence. Well, it's not exactly a big problem or anything. Personally speaking I found it fairly easily understood, but it doesn't justify itself very well.